modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentLocalServiceImpl.java

```diff
@@ -282,6 +282,27 @@ public int getCommerceShipmentsCount(
 			findCommerceShipmentStatusesByCommerceOrderId(commerceOrderId);
 	}
 
+	@Indexable(type = IndexableType.REINDEX)
+	@Override
+	public CommerceShipment reprocessShipment(long commerceShipmentId)
+		throws PortalException {
+
+		CommerceShipment commerceShipment =
```

Comment:
@riccardo-alberti  This means that ther eis no need of validation and from any status to can change back to SHIPMENT_STATUS_PROCESSING?